### DISCUSSION TITLE
Macos installation takes too long

### DISCUSSION THREAD

**Discussion by georgerabus** [quickemu-project/quickemu#1507] [Quickemu]
![image](https://github.com/user-attachments/assets/99a59f0c-fd89-4b38-ad3f-4dfdd6ce40ab)
I am currently waiting for it to install to disk, at first it was saying 2 hours left, now 9 hours, and I don't see the time to change. Is it normal to be this long? doesnt make sense to me. Windows installation takes like 5 minutes on my ssd

**Comment by halfninja** [comment]
Does the log say anything (Window > Installer Log)

**Comment by carissab96** [comment]
I'm having the same issue with Ventura. It was 2 hours and 17 minutes and is now 12 hours and 59 minutes...It's been running for about 30 minutes but does seem to be moving down now instead of up. I read somewhere that the size of assigned vram in the .conf file can sometimes cause the installer to stall out. I was going to let the installation run for a while and see what happens before I cancel the installation and adjust the .conf file to a lower ram amount. The comment was on an article written by Nick Sherlock where the user reduced the ram to 4gb in the configuration file and the install took about 20 minutes...If I get to that point, I'll update this comment and let you know if it really works.
